apps/src/music/player/MusicLibrary.ts

```diff
@@ -298,6 +337,43 @@ export const LibraryValidator: ResponseValidator<LibraryJson> = response => {
   return libraryJson;
 };
 
+const localizeLibrary = (
+  library: LibraryJson,
+  translations: Translations
+): LibraryJson => {
+  const libraryJsonLocalized = JSON.parse(
+    JSON.stringify(library)
+  ) as LibraryJson;
```

Comment:
Is this just to do a deep copy of the object?

If so, it's quite slow (there are also gotchas around using this as a general technique with contents that can't be serialized to json, but that doesn't apply here). We're already using `lodash` - `cloneDeep` is better suited here.

```
import cloneDeep from 'lodash/cloneDeep';
const libraryJsonLocalized = cloneDeep(libraryJson)
```